plugin.xml

```diff
@@ -10,22 +10,48 @@
     <keywords>cordova,urbanairship</keywords>
     <repo>https://github.com/urbanairship/urbanairship-accengage-cordova.git</repo>
     
+    <engines>
+        <engine name="cordova" version=">=9.0.0"/>
```

Comment:
You should probably specify the versions specified in our `urbanairship-cordova` `plugin.xml`